### PR TITLE
Update Chromium to version 119.0.6045.105

### PR DESCRIPTION
**Pull request by vlad-lubenskyi**
In this changeset, we upgrade Chromium to version 119.0.6045.105.

Please review the modified Chromium licenses.

### CODE REVIEW

File: chromium-licenses/third_party/dawn/third_party/dxc/LICENSE.TXT [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+WILL ANY COPYRIGHT HOLDER, OR ANY OTHER PARTY WHO MAY MODIFY AND/OR
+REDISTRIBUTE THE PROGRAM AS PERMITTED ABOVE, BE LIABLE TO YOU FOR DAMAGES,
+INCLUDING ANY GENERAL, SPECIAL, INCIDENTAL OR CONSEQUENTIAL DAMAGES ARISING
+OUT OF THE USE OR INABILITY TO USE THE PROGRAM (INCLUDING BUT NOT LIMITED
+TO LOSS OF DATA OR DATA BEING RENDERED INACCURATE OR LOSSES SUSTAINED BY
+YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY OTHER
+PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGES.
+
+Autoconf Exception
```
Comment (vladimir-ikryanov) [inline chromium-licenses/third_party/dawn/third_party/dxc/LICENSE.TXT:574]: This license refers GPL. We need to take a look at this "Autoconf Exception" and make sure that license cannot be applied to our product.

**Comment by vladimir-ikryanov** [comment]
Here's the result of my research regarding GPL license in the Autoconf Exception text.

The **Autoconf Exception** text in `chromium-licenses/third_party/dawn/third_party/dxc/LICENSE.TXT` is the following:

```
Autoconf Exception

As a special exception, the Free Software Foundation gives unlimited
permission to copy, distribute and modify the configure scripts that are the
output of Autoconf. You need not follow the terms of the GNU General Public
License when using or distributing such scripts, even though portions of the
text of Autoconf appear in them. The GNU General Public License (GPL) does
govern all other use of the material that constitutes the Autoconf program.

Certain portions of the Autoconf source text are designed to be copied (in
certain cases, depending on the input) into the output of Autoconf. We call
these the "data" portions. The rest of the Autoconf source text consists of
comments plus executable code that decides which of the data portions to
output in any given case. We call these comments and executable code the "non-
data" portions. Autoconf never copies any of the non-data portions into its
output.

This special exception to the GPL applies to versions of Autoconf released by
the Free Software Foundation. When you make and distribute a modified version
of Autoconf, you may extend this special exception to the GPL to apply to your
modified version as well, *unless* your modified version has the potential to
copy into its output some of the text that was the non-data portion of the
version that you started with. (In other words, unless your change moves or
copies text from the non-data portions to the data portions.) If your
modification has such potential, you must delete any notice of this special
exception to the GPL from your modified version.
```

The component where the license with this text was found is DirectX Shader Compiler (DXC). The project is a fork of the [LLVM](http://llvm.org/) and [Clang](http://clang.llvm.org/) projects, modified to accept HLSL and emit a validated container that can be consumed by GPU drivers.

This component has a directory with the `autoconf\config.guess` script inside which is under GPL license. Inside this script I see the following license:

```
# This file is free software; you can redistribute it and/or modify it
# under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
# General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street - Fifth Floor, Boston, MA
# 02110-1301, USA.
#
# As a special exception to the GNU General Public License, if you
# distribute this file as part of a program that contains a
# configuration script generated by Autoconf, you may include it under
# the same distribution terms that you use for the rest of that program.
```

This script is a bash/shell program that does an attempt to guess a canonical system name. I suppose the DXC might invoke this program to get a canonical system name.

**We don't modify this script, don't distribute it or don't link with it.**

@oleksandra-kovalenko Is it enough to be sure that this license is not applicable?

**Comment by vlad-lubenskyi** [comment]
@oleksandra-varavina

**Comment by oleksandra-varavina** [comment]
_>This component has a directory with the autoconf\config.guess script inside which is under GPL license._

If this directory **is not** part of the DirectX Shader Compiler (DXC) component in our Chromium build, then we can modify this license file and remove the reference to the autoconf license.

If the mentioned directory **is** part of the build, can we cut it out?

**Comment by vladimir-ikryanov** [comment]
@oleksandra-varavina this directory is part of the DirectX Shader Compiler (DXC) and we cannot remove it. This directory contains only one bash/shell script that is used during Chromium build. This script is not included into the final JxBrowser/DotNetBrowser build. It's just an executable program. We don't modify it and don't distribute it. Let's just reference the Autoconf exception in our table because it's applicable in our case.

File: chromium-licenses/third_party/dawn/third_party/dxc/LICENSE.TXT (hunk 1 above)
Comment (vladimir-ikryanov) [reply chromium-licenses/third_party/dawn/third_party/dxc/LICENSE.TXT:574]: See [discussion](https://github.com/TeamDev-IP/Chromium-Licences/pull/47).

**Comment by oleksandra-varavina** [comment]
Thank you. We will proceed with creating the new table and we will reference "Autoconf Exception" as a comment for this specific component.

**Review by oleksandra-varavina — APPROVED** [review]
Licenses checked, issues resolved.
Table creation is in progress.

**Pull request merged**